init/arch_specs/eessi_arch_arm.spec

```diff
@@ -1,9 +1,12 @@
 # ARM CPU architecture specifications (see https://gpages.juszkiewicz.com.pl/arm-socs-table/arm-socs.html for guidance)
 # CPU implementers: 0x41 (ARM), 0x46 (Fujitsu) - also see https://github.com/hrw/arm-socs-table/blob/main/data/socs.yml
+# Flipping the order of nvidia/grace and google/axion results in nvidia/grace being the "top" on Axion systems
```

Comment:
@TopRichard I don't think we're doing anything special here for either grace or Axion.

When CPU target A is a subset of CPU target B, then A should be listed before B to ensure a correct order produced by `archdetect`.

That's also the case for `neoverse_n1` vs `neoverse_v1`, for example.

So maybe this should be worded a bit more generally, to point out the importance of the order below without making it specific to particular CPU targets?